pkg/server/events.go

```diff
@@ -106,53 +107,20 @@ func (em *eventMonitor) handleEvent(evt *events.Envelope) {
 		e := any.(*eventtypes.TaskExit)
```

Comment:
Do we not handle sandbox with TaskOOM?

Reply:
Yeah, we don't need to do that. Even we handle, what can we do? :)